public/app/plugins/datasource/prometheus/querybuilder/components/LabelFilters.test.tsx

```diff
@@ -0,0 +1,133 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
These tests are copy paste as well from the shared. Can we add some test coverage for the new functionality?

Reply:
I tried and mocking the autocomplete functionality in these components was not trivial, could we follow up with adding some e2e tests on this behavior?

Reply:
Uck, I'd like to get some more test coverage on this, how about testing the updated variable interpolation in the PromQueryBuilder.test.tsx?